### DISCUSSION TITLE
Error dlm aplikasi web dashboard - Capstone Flask Dashboard UI

### DISCUSSION THREAD

**Discussion by b-th** [teamalgoritma/community#832] [Q&A] [answered]
Saat saya run link yg di generate setelah menjalankan python app.py pada terminal, hasilnya error 
<img width="1424" alt="Screenshot 2023-12-18 at 03 04 42" src="https://github.com/teamalgoritma/community/assets/154104693/011f1e88-c3f6-499b-84b5-aa97e3eb8888">
<img width="900" alt="Screenshot 2023-12-18 at 03 08 20" src="https://github.com/teamalgoritma/community/assets/154104693/75634142-0cb4-4f05-9cbe-f455564ae509">

mohon bantuannya

**Comment by arkanayudhistira** [comment, marked as answer]
Dear Ibu @b-th,

Berdasarkan keterangan errornya, kemungkinan besar error tersebut terjadi pada fungsi `playstore.groupby('category').agg...` dimana ibu menggunakan nama "Total" untuk `sort_values()`. Namun, pada dataframe playstore hasil agregasi tidak terdapat kolom total yang akan disorting. Silahkan coba untuk menambahkan code berikut sebelum melakukan sorting :
```
.rename({'Category':'Total'}
```
Untuk mengganti nama kolom `Category` hasil agregasi menjadi `Total`

Semoga membantu!
